docs/GroupDefs.hpp

```diff
@@ -159,155 +164,485 @@
  * Charm++
 
 SpECTRE builds a layer on top of Charm++ that performs various safety checks and
-initialization for the user that can otherwise lead to difficult to debug
+initialization for the user that can otherwise lead to difficult-to-debug
 undefined behavior. The central concept is what is called a %Parallel
 Component. A %Parallel Component is a struct with several type aliases that
 is used by SpECTRE to set up the Charm++ chares and allowed communication
-patterns. It might be most natural to think of %Parallel Components
-as input arguments to a metaprogram that writes the parallelization
-infrastructure that you requested for the executable. There is no restriction
-on the number of %Parallel Components, though practically it is best to have
-around 10 at most.
-
-Each %Parallel Component must have the following type aliases:
-1. `using chare_type` is set to one of `Parallel::Algorithms::Singleton`,
-   `Parallel::Algorithms::Array`, `Parallel::Algorithms::Group`, or
-   `Parallel::Algorithms::Nodegroup`. What these mean is explained below.
+patterns. %Parallel Components are input arguments to the compiler, which then
+writes the parallelization infrastructure that you requested for the executable.
+There is no restriction on the number of %Parallel Components, though
+practically it is best to have around 10 at most.
+
+### The Metavariables Class
+
+SpECTRE takes a different approach to input options passed to an executable than
+is common. SpECTRE not only reads an input file at runtime but also has many
+choices made at compile time. The compile time options are specified by what is
+referred to as the metavariables. What exactly the metavariables specifies
```

Comment:
_the metavariables specify_ or _the metavariables struct specifies_

Reply:
Going with the second since metavariables is singular so the first is not correct :)